### ISSUE TITLE
Color scheme options have insufficient color contrast

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#16]
**Bright**

- [x] Link color #2ec4b6 on a default white page background (contrast ratio 2.17)

**Comment by andrewmarino** [comment]
https://github.com/unitymakesus/justgetsimple/commit/11b39f6d2c238c7f47b07c36c2aa1cee5a4289e0
